src/librustdoc/html/static/rustdoc.css

```
@@ -427,9 +427,13 @@ nav.sub {
 	border-bottom: 1px solid;
 }
 
-#main > .docblock h1 { font-size: 1.3em; }
-#main > .docblock h2 { font-size: 1.15em; }
-#main > .docblock h3, #main > .docblock h4, #main > .docblock h5 { font-size: 1em; }
+.top-doc .docblock h1 { font-size: 1.3em; }
+.top-doc .docblock h2 { font-size: 1.15em; }
+.top-doc .docblock h3,
+.top-doc .docblock h4,
+.top-doc .docblock h5 {
```

I think it should instead be instead:

```suggestion
#main > details > .docblock h1 { font-size: 1.3em; }
#main > details > .docblock h2 { font-size: 1.15em; }
#main > details > .docblock h3,
#main > details > .docblock h4,
#main > details > .docblock h5 {
```

---

Once #85280 merges, `.top-doc` will only represent the "top" docblock wrapper, so I think this will be correct after that merge.

---

Ok, let's wait for it then.